tools/st2ctl

```diff
@@ -231,6 +231,13 @@ function getpids(){
       done
     else
       echo "${COM} is not running"
+      if [ "$COM" == "st2sensorcontainer" ] || [ "$COM" == "sensor_container" ]
+      then
+        echo "Sensor container did not detect valid sensors to run."
+        echo "Please install sensors or check sensors configuration for existing ones."
+        echo "See http://docs.stackstorm.com/sensors.html."
+        echo "Use 'st2ctl restart-component st2sensorcontainer' to restart after you fix the problems."
```

Comment:
There is no straight forward way to get the base_pack info in this script. Other than that, this comment seems fine.